Catalyst/source/index.rst

```diff
@@ -28,11 +28,11 @@ The main documentation for the site is organized into a couple sections:
 
    gettingstarted
    menu-options
-   settings

```

Comment:
Is "settings" removed? Why?

Reply:
> Is "settings" removed? Why

In main dashboard data under settings tab called as "workflow".all org settings,devops roles all are under workflow tab that'y changed as workflow.